packages/core/useTitle/index.ts

```diff
@@ -18,11 +18,11 @@ export interface UseTitleOptions extends ConfigurableDocument {
    *
    * @default '%s'
    */
-  titleTemplate?: MaybeRef<string> | ((title: string) => string)
+  titleTemplate?: MaybeComputedRef<string> | ((title: string) => string)
 }
 
 export function useTitle(
-  newTitle?: MaybeRef<string | null | undefined>,
+  newTitle?: MaybeComputedRef<string | null | undefined>,
```

Comment:
This one is indented (it's an overload). `MaybeComputedRef` should only be used when we are sure we only read the value (in this case it's retuned as a mutable ref)